<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QLabel" name="labelLogin_Seach">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLogin_Seach">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValid_Seach">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valid</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QPushButton" name="pushButtonValid_GetAll">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valid</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPage_GetAll">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxElement_GetAll">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelPage_GetAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelElement_GetAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Éléments</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QPushButton" name="pushButtonValid_AddUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valid</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditConfirm_AddUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword_AddUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLogin_AddUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelLogin_AddUser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassword_AddUser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelConfirm_AddUser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm
Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLineEdit" name="lineEditOldPassword_Edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNewLogin_Edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNewPassword_Edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditConfirm_Edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValid_Edit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valid</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOldPassword_Edit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Old
Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNewLogin_Edit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New
Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNewPassword_Edit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New
Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelConfirm_Edit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm
Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QLineEdit" name="lineEditPassword_Delete">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValid_Delete">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valid</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassword_Delete">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page6">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Opération éfféctuée</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGetAll">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearch">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
